core/test/utils/file_utils_test.dart

```diff
@@ -38,6 +41,58 @@ void main() {
 
       file.delete();
     });
+
+    test(
+      'Should delete file '
+      'when deleteCompressedFileOnMobile is called '
+      'and file exists '
+      'and Platform is mobile',
+    () async {
+      // arrange
+      final file = await FileUtils().saveToFile(nameFile: fileName, content: fileContent);
+
+      // act
+      await FileUtils().deleteCompressedFileOnMobile(file.path, pathContains: fileName);
+
+      // assert
+      expect(await file.exists(), equals(false));
+    });
+
+    test(
+      'Should not delete file '
+      'when deleteCompressedFileOnMobile is called '
+      'and file exists '
+      'and Platform is not mobile',
+    () async {
+      // arrange
+      debugDefaultTargetPlatformOverride = TargetPlatform.macOS;
+      final file = await FileUtils().saveToFile(nameFile: fileName, content: fileContent);
+      expect(await file.exists(), equals(true));
+
+      // act
+      await FileUtils().deleteCompressedFileOnMobile(file.path, pathContains: fileName);
+
+      // assert
+      expect(await file.exists(), equals(true));
+      debugDefaultTargetPlatformOverride = null;
+    });
+
+    test(
+      'Should not delete file '
+      'when deleteCompressedFileOnMobile is called '
+      'and file does not exist '
+      'and Platform is mobile',
+    () async {
+      // arrange
+      final file = File('$fileName.txt');
+      expect(await file.exists(), equals(false));
+
+      // act
+      await FileUtils().deleteCompressedFileOnMobile(file.path, pathContains: fileName);
+
+      // assert
+      expect(await file.exists(), equals(false));
```

Comment:
This test case makes no sense.

Reply:
If the method is called on a non-existent file, nothing will happen.

Reply:
The last `expect` is meaningless. How do we check to make sure that if the file doesn't exist, the function won't be called. or return a failure value.

Reply:
Test deleted